docs/appkit/migration/from-connectkit-next.mdx

```diff
@@ -13,20 +13,20 @@ To migrate from ConnectKit to Reown AppKit, please follow the steps below.
 
 ## Step 1. Create a project in Reown Cloud
 
-+ Create a new project on [Reown Cloud](https://cloud.reown.com) and obtain a new project ID.
+- Create a new project on [Reown Cloud](https://cloud.reown.com) and obtain a new project ID.
 
 ## Step 2. Install & uninstall libraries
 
-+ Run this command to install Reown AppKit and uninstall ConnectKit.
+- Run this command to install Reown AppKit and uninstall ConnectKit.
 
 ```bash npm2yarn
 npm install @reown/appkit @reown/appkit-adapter-wagmi && npm uninstall connectkit
 ```
 
 ## Step 3. Change the code in the `/components/Web3Provider.tsx` file
 
-+ Navigate to the `/components/Web3Provider.tsx` file inside your ConnectKit example Web3 app directory. 
-+ Now, you need to remove the existing configuration that uses ConnectKit and replace it with Reown AppKit. Refer to the code snippet below.
+- Navigate to the `/components/Web3Provider.tsx` file inside your ConnectKit example Web3 app directory.
+- Now, you need to remove the existing configuration that uses ConnectKit and replace it with Reown AppKit. Refer to the code snippet below.
```

Comment:
@enesozturk I think prettier messed up the formatting here. Can you please revert this back and update the PR?